dora/core/client/fs/src/main/java/alluxio/client/file/ufs/UfsBaseFileSystem.java

```diff
@@ -116,6 +116,29 @@ public UfsBaseFileSystem(FileSystemContext fsContext, UfsFileSystemOptions optio
     LOG.debug("Creating file system connecting to ufs address {}", ufsAddress);
   }
 
+  /**
+   * Constructs a wrapper file system based on given UnderFileSystem.
+   *
+   * @param fsContext file system context
+   * @param options the ufs file system options
+   * @param ufs the under file system
+   */
+  public UfsBaseFileSystem(FileSystemContext fsContext, UfsFileSystemOptions options,
```

Comment:
can we share the create logics between the two constructors? since most of the logics are the same